load-test/cantabular/autocannon-go-2/INSTRUCTIONS.md

```diff
@@ -0,0 +1,155 @@
+# Load testing Cantabular Publishing servers in Staging to exercise its ASG
+
+The files in this folder `autocannon-go-2` have been derived from:
+
+<https://github.com/GlenTiki/autocannon-go>
+
+NOTE: the original files build and run, but doing `go mod tidy` throws up an issue ... which persists in the derived code.
+
+## Purpose
+
+The derived code is tunned to make sufficient requests to the Cantabular load balancer endpoint for enough time to see close to 100% CPU usage that will cause the ASG to add more servers to its ASG.
+
+The new code cyles through a list of codebook endpoints to get the Cantabular servers to load all of the data from the databases into memory to achieve maximum utilisation / performance from the Cantabular servers as advised by Sensible code.
+
+## Pre-requsites
+
+First INFORM your team and anyone else affected by these load tests that they will be taking place !
+
+The Cantabular servers will need to have the ansible code loading up the following list of files (in file ansible/inventories/staging/group_vars/all):
+
```

Comment:
do we have to mention that these file are just examples, and need to be replaced with the one's that is used for testing purpose?